.travis.yml

```diff
@@ -39,12 +39,16 @@ env:
 before_install:
   - yes | sdkmanager "platforms;android-27"
 
-script:
-- echo "Travis branch is $TRAVIS_BRANCH"
-- echo "Travis branch is in pull request $TRAVIS_PULL+REQUEST  buildNumber $TRAVIS_BUILD_NUMBER"
-- ./gradlew app:googleServicesDecrypt assembleDebug -PminSdk=$MIN_SDK
-- ./gradlew lint ktlint checkstyleMain --continue -x app:ktlint
-- ./gradlew cleanTest test -Pusecache=true -PparallelForks=2
+jobs:
```

Comment:
Travis does not share compiled data between nor android sdk setup between stages, it uses a new fresh VM for each stage.

With this configuration the CI will took even more time, and also will waste travis resources.